model/model.go

```diff
@@ -35,7 +35,7 @@ type Domain interface {
 	// Min of a domain and a boolean indicating it is nonempty.
 	Min() (int, bool)
 	// Remove values from a domain.
-	Remove(...int) Domain
+	Remove([]int) Domain
```

Comment:
I left a similar comment in the plugins counterpart, but can't we change it to keep the syntax from line 64?

```go
Remove(int, ...int) Domain
```

And the syntax for `Domains` is the same

```go
Remove(int, ...int) Domains
```